presentation.py

```diff
@@ -99,18 +92,26 @@ def add_text(slide, text):
     p.text = text
 
 def apply_layout(slide, layout):
-    if 'centerTitle' in layout:
-        add_center_title(slide, layout['centerTitle'])
-    if 'topTitle' in layout:
-        add_top_title(slide, layout['topTitle'])
-    if 'sideTitle' in layout:
-        add_side_title(slide, layout['sideTitle'])
-    if 'img' in layout:
-        add_img(slide, layout['img'])
-    if 'sideImg' in layout:
-        add_side_img(slide, layout['sideImg'])
-    if 'text' in layout:
-        add_text(slide, layout['text'])
+    if 'stackDirection' in layout and layout['stackDirection'] == 'horizontal':
+        direction = Direction.HORIZONTAL
+        if 'title' in layout:
+            add_title(slide, layout['title'], direction)
+        if 'centerTitle' in layout:
+            add_center_title(slide, layout['centerTitle'])
+        if 'img' in layout:
+            add_img(slide, layout['img'], direction)
+        if 'text' in layout:
+            add_text(slide, layout['text'])
+    else:
+        direction = Direction.VERTICAL
+        if 'title' in layout:
+            add_title(slide, layout['title'], direction)
+        if 'centerTitle' in layout:
+            add_center_title(slide, layout['centerTitle'])
+        if 'img' in layout:
+            add_img(slide, layout['img'], direction)
+        if 'text' in layout:
+            add_text(slide, layout['text'])
```

Comment:
This if-else statement is a bit redundant.

I guess the code below also works:
```python
    if 'stackDirection' in layout and layout['stackDirection'] == 'horizontal':
        direction = Direction.HORIZONTAL
    else:
        direction = Direction.VERTICAL

    if 'title' in layout:
        add_title(slide, layout['title'], direction)
    if 'centerTitle' in layout:
        add_center_title(slide, layout['centerTitle'])
    if 'img' in layout:
        add_img(slide, layout['img'], direction)
    if 'text' in layout:
        add_text(slide, layout['text'])
```

Reply:
It would be great if we can define a function like `parse_style(style_of_dictionary)` and get a `Style` object from this, then pass it to `add_title` and `add_image` as an argument instead of `direction`.